libethereum/Block.cpp

```diff
@@ -104,7 +104,7 @@ void Block::resetCurrent()
 	m_transactionSet.clear();
 	m_currentBlock = BlockInfo();
 	m_currentBlock.setCoinbaseAddress(m_beneficiary);
-	m_currentBlock.setTimestamp(max(m_previousBlock.timestamp() + 1, (u256)time(0)));
```

Comment:
The cast here is needed. It won't implicitly convert to u256 for the max() function.